src/main/scala/mesosphere/util/state/zk/ZKStore.scala

```diff
@@ -6,18 +6,21 @@ import com.fasterxml.uuid.impl.UUIDUtil
 import com.google.protobuf.{ ByteString, InvalidProtocolBufferException }
 import com.twitter.util.{ Future => TWFuture }
 import com.twitter.zk.{ ZNode, ZkClient }
+import mesosphere.marathon.io.IO
 import mesosphere.marathon.{ Protos, StoreCommandFailedException }
 import mesosphere.util.ThreadPoolContext
 import mesosphere.util.state.zk.ZKStore._
 import mesosphere.util.state.{ PersistentEntity, PersistentStore }
 import org.apache.log4j.Logger
 import org.apache.zookeeper.KeeperException
-import org.apache.zookeeper.KeeperException.{ NodeExistsException, NoNodeException }
+import org.apache.zookeeper.KeeperException.{ NoNodeException, NodeExistsException }
 
 import scala.concurrent.duration.Duration
 import scala.concurrent.{ Await, Future, Promise }
 
-class ZKStore(val client: ZkClient, rootNode: ZNode) extends PersistentStore {
+case class CompressionConf(enabled: Boolean, sizeLimit: Long)
```

Comment:
I don't want to use ScallopConf objects other than setup logic, mainly to make testing code easy.